cfgov/unprocessed/css/organisms/card-group.less

```diff
@@ -1,47 +1,42 @@
 .o-card-group {
-    padding: unit( 30px / @base-font-size-px, em );
 
-    &_cards {
-        display: flex;
-        flex: 1;
+    &__column-2 &_cards {
+        grid-template-columns: 1fr 1fr;
+    }
 
-        & .m-card {
-            margin-left: unit( 20px / @base-font-size-px, em );
+    &__column-3 &_cards {
+        grid-template-columns: repeat( 3, minmax( 0, 1fr ) );
 
-            .lt-ie10 & {
-                display: block;
-                width: 32%;
-                margin-left: 2%;
-                float: left;
-            }
+        // Convert to 2-column layout at small screen sizes;
+        .respond-to-max( @bp-med-min, {
+            grid-template-columns: 1fr 1fr;
+        } );
+    }
 
-            &:first-child {
-                margin-left: 0;
-            }
-        }
+    &_cards {
+        display: grid;
+        grid-column-gap: unit( 20px / @base-font-size-px, em );
+        grid-row-gap: unit( 20px / @base-font-size-px, em );
 
         .respond-to-max( @bp-xs-max, {
-            flex-direction: column;
-
-            & .m-card {
-                margin-top: unit( @grid_gutter-width / @base-font-size-px, em );
-                margin-left: 0;
-                // Overrides flex-basis:1 to prevent card content from overflowing container
-                // when flex-direction is set to column at small screen sizes
-                flex-basis: auto;
-            }
-
-            & .m-card:first-child {
-                margin-top: 0;
-            }
+            // Make sure cards are stacked at mobile size.
+            grid-template-columns: 100% !important;
         } );
     }
 
     &__bg-green {
+        padding: unit( 30px / @base-font-size-px, em );
         background: @green-20;
     }
+}
+
+// Hack for Edge/IE to convert grid to block layout.
+// See https://jeffclayton.wordpress.com/2015/04/07/css-hacks-for-windows-10-and-spartan-browser-preview/
+// Necessitated because of https://stackoverflow.com/questions/46743722/items-in-css-grid-in-ie-superimposed-on-each-other
+_:-ms-lang(x),.o-card-group_cards {
+    display: block !important;
```

Comment:
I'm not necessarily opposed, but I am wondering whether it makes sense to convert the cards to grid if that necessitates block display on all IE. It looks like IE11 is one of the most commonly used browsers on our site, so we'd be serving the stacked display to a fairly large group of users. Is there a workaround we could use instead of block?

Reply:
It's a good point, IE11 is about 10% of visitors. My thinking here was that this is the kind of layout grid was made for so we get a lot of flexibility with a little code. We certainly could expand on the fallbacks for IE in the above block, but I didn't think it looked terrible as is, so that extra maintenance burden may not be worth it. Flexbox is an option too, but I was worried about unexpected wrapping across the breakpoints.

<img width="1224" alt="Screen Shot 2020-03-03 at 5 43 04 PM" src="https://user-images.githubusercontent.com/704760/75827306-54000400-5d77-11ea-87f8-fd17b46b63b4.png">

Of course it matters if the stakeholders aren't happy with that change, but I'm not sure who has say on this page? cc @schaferjh

Reply:
I agree that it would be really nice to use grid here and I don't think that the Ask cards look too bad stacked either. I wonder if there's more visual impact with the three columns of cards stacking on the home page, though.

Reply:
How about this? I put in a floated layout fallback in https://github.com/cfpb/cfgov-refresh/pull/5538/commits/6cd9abd3a6d3f50bae90609066a3d84dcf792d4f

The only issue is the boxes aren't equal height and there's slightly more margin on the right. I'm not sure how else to do a fallback without putting each row into its own container. What do you think?

<img width="1148" alt="Screen Shot 2020-03-05 at 1 37 48 PM" src="https://user-images.githubusercontent.com/704760/76015787-bc6ef280-5ee9-11ea-8b41-05b3e5c30e9f.png">

<img width="1235" alt="Screen Shot 2020-03-05 at 1 37 09 PM" src="https://user-images.githubusercontent.com/704760/76015796-c264d380-5ee9-11ea-8ae2-37f48c027b0b.png">

<img width="1228" alt="Screen Shot 2020-03-05 at 1 37 30 PM" src="https://user-images.githubusercontent.com/704760/76015809-c5f85a80-5ee9-11ea-966c-c213ad61328d.png">

Reply:
This definitely looks better to me. I defer to @huetingj, but I would be fine with merging this to get the fix out for Ask. We can always revisit the fallback in the design system if necessary.

Reply:
:+1: from me to merging this side-by-side layout.